plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestingSnowflakeServer.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.snowflake;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+import static java.util.Objects.requireNonNull;
+
+public class TestingSnowflakeServer
+        implements Closeable
+{
+    public static final String TEST_URL = requireNonNull(System.getProperty("connection-url"), "connection-url is not set");
+    public static final String TEST_USER = requireNonNull(System.getProperty("connection-user"), "connection-user is not set");
+    public static final String TEST_PASSWORD = requireNonNull(System.getProperty("connection-password"), "connection-password is not set");
+    public static final String TEST_DATABASE = requireNonNull(System.getProperty("snowflake.database"), "snowflake.database is not set");
+    public static final String TEST_SCHEMA = "tpch";
+    public static final String TEST_WAREHOUSE = requireNonNull(System.getProperty("snowflake.warehouse"), "snowflake.warehouse is not set");
+    public static final String TEST_ROLE = requireNonNull(System.getProperty("snowflake.role"), "snowflake.role is not set");
+
+    private final Connection conn;
+
+    public TestingSnowflakeServer()
+    {
+        try (Connection connection = DriverManager.getConnection(TEST_URL, getProperties());
+                Statement statement = connection.createStatement()) {
+            this.conn = connection;
+        }
```

Comment:
Move to `execute()` method.

Reply:
This part I wrote in Constructor of `TestingSnowflakeServer()` and to initialize `private final Connection conn;`. If I move this to `execute()` method, what to do for the `conn` et `close()` pls ?

Reply:
We don't need `close()` in this class. Please stop implementing `Closeable`.